.github/workflows/janus-ci.yml

```diff
@@ -152,3 +152,24 @@ jobs:
         ./configure $JANUS_CONFIG_COMMON $JANUS_CONFIG_OPTS
          make -j$(nproc)
          make check-fuzzers
+  javascript-lint:
+    runs-on: ubuntu-20.04
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install modules
+      run: npm install --only=dev
+    - name: Run ESLint
+      run: node_modules/.bin/eslint html/janus.js
+  javascript-rollup:
```

Comment:
imho I don't get the rationale of this.
Could you please elaborate ?

Reply:
It was asked whether the code would still build into a module. This confirms that rollup process still completes (although does not check that the resulting file works.)

Reply:
I totally forgot about the `npm` folder we have in the repo, my bad.
Anyway now that I have noticed it, I guess we should also:
- enable strict mode [in the rollup config](https://github.com/meetecho/janus-gateway/blob/master/npm/rollup.config.js#L8) 
- not sure, but maybe we can now [enable the linter again](https://github.com/meetecho/janus-gateway/blob/master/npm/module.js#L1) in the generated module ?

Reply:
Strict mode is now enabled (by removing the config disabling it.) We end up with `use strict` twice in the CJS file now, but I believe that is harmless.

The generated module has this comment, ` * Since this counts as 'autogenerated' code, ESLint is instructed to ignore the contents of this file when linting your project.` which I guess means you're not expected to re-lint rolled-up modules.

Reply:
That comment is already there before the module generation.

Reply:
I'm not sure what is the best practice here honestly

Reply:
> We end up with use strict twice in the CJS file now, but I believe that is harmless
Oh right... sorry for the noise can you revert this, I guess we can proceed with the merge then.

Reply:
I've reverted the change to `config.rollup.js`.